setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
I changed it locally, the requirements files that it generated included the invenio-db[versioning], but not invenio-db[mysql] (or postgresql) nor the pymysql or psycopg2 packages